### ISSUE TITLE
API Server fails after Update from 0.5.3-alpha.6 to version 0.5.3-alpha.9

### ISSUE THREAD

**Issue by kiguigui** [revoltchat/delta#121]
After changing docker image tag from 0.5.3-alpha.6 to version 0.5.3-alpha.9. 

Logs entries:

```
[2021-11-23T08:45:07Z` INFO  revolt] Starting REVOLT server [version 0.5.3-alpha.9].
```
```
[2021-11-23T08:45:07Z` INFO  revolt::database::migrations::scripts] Starting database migration.
```
```
[2021-11-23T08:45:07Z` INFO  revolt::database::migrations::scripts] Running migration [revision 10 / 2021-11-01]: Remove nonce values on channels and servers.
```
```
[2021-11-23T08:45:07Z` INFO  revolt::database::migrations::scripts] Running migration [revision 11 / 2021-11-14]: Add indexes to database.
```

```
thread` 'main' panicked at 'Failed to create server_members index.: Error { kind: CommandError(CommandError { code: 86, code_name: "IndexKeySpecsConflict", message: "An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, key: { _id.server: 1, _id.user: 1 }, name: \"compound_id\" }, existing index: { v: 2, key: { _id.channel: 1, _id.user: 1 }, name: \"compound_id\" }", labels: [] }), labels: [] }', src/database/migrations/scripts.rs:444:10`
```

**Comment by insertish** [comment]
Fixed in 788580a64f756c97cfccdbd23bc8eeedc5a270a3.

To fix locally you need to manually edit the database: (ideally with MongoDB Compass)
- Edit the only document in `migrations` and set revision to `12`.
- Enter shell mode and run:
  ```
  use revolt;
  db.channel_unreads.createIndex({ "_id.channel": 1, "_id.user": 1 });
  db.channel_unreads.createIndex({ "_id.user": 1 });
  db.server_members.createIndex({ "_id.server": 1, "_id.user": 1 });
  db.server_members.createIndex({ "_id.user": 1 });
  ```

**Comment by kiguigui** [comment]
that worked, thank you for your quick response :)
